app/src/main/res/layout/activity_login.xml

```diff
@@ -48,13 +50,30 @@
         android:layout_height="60dp"
         android:layout_marginStart="128dp"
         android:layout_marginEnd="128dp"
-        android:layout_marginBottom="-50dp"
+        android:layout_marginBottom="8dp"
         android:backgroundTint="@color/TG_blue"
         android:fontFamily="@font/poppins_bold"
         android:text="@string/login_register_btn"
         android:textAllCaps="false"
         android:textColor="@color/white"
         app:cornerRadius="24dp"
+        app:layout_constraintBottom_toTopOf="@+id/loginGoogleBtn"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
```

Comment:
You can replace `Button` with `com.google.android.gms.common.SignInButton` so the component looks more standard for Google sign in
<img width="427" alt="Screenshot 2023-11-24 at 12 54 40 pm" src="https://github.com/Gopher-Industries/Guardian/assets/66286752/e2edc0a1-53ec-4501-ae8b-a900b1f9563b">

Reply:
Seems like a good idea, changed!